docs/rules/filename-naming-convention.md

```diff
@@ -98,6 +121,28 @@ module.exports = {
 };
 ```
 
+##### `ignorePattern`
+
+This can be usefull when enforcing rules using group capture to ignore some files. For example if you have a folder and you want all the files inside to match the folder name except the index file you could have a rule set up that way :
+
+```js
+module.exports = {
+  plugins: ['check-file'],
+  rules: {
+    'check-file/filename-naming-convention': [
+      'error',
+      {
+        '**/**/*': '<1>*',
+      },
+      {
+        ignoreMiddleExtensions: true,
+        ignorePattern: 'index.*',
+      },
+    ],
+  },
+};
+```
+
```

Comment:
There is no necessary to introduce a new parameter `ignorePattern`, you can use a glob expression like `src/*/!(index).*` to select target files. [see more](https://globster.xyz/?q=src%2F*%2F!(index).*&f=src%2FPortal%2FPortal.test.tsx%2Csrc%2FPortal%2FPortal.tsx%2Csrc%2FPortal%2FP.py%2Csrc%2FPortal%2FPortal.types.tsx%2Csrc%2FPortal%2Findex.tsx%2Csrc%2FPortal%2Finde.css)

Reply:
Okey, yes it's even better . Thanks